src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/VisualStudioFinder.cs

```diff
@@ -35,35 +35,34 @@ public void Configure(WorkspaceOptions options)
 
         private (string? Path, int? Version) GetLatestVisualStudioPath(string? suppliedPath)
         {
-            var latest = GetLatestPath(suppliedPath);
+            var latest = GetLatestVisualStudio(suppliedPath);
 
-            if (latest is null)
+            if (latest.InstallPath is null)
```

Comment:
Seems like latest could be null so this should probably be latest?.InstallPath...

Reply:
it cannot be null anymore - it now always returns a non-null object. Only latest.InstallPath can be null.